pandas/io/excel.py

```diff
@@ -417,10 +419,13 @@ def _parse_cell(cell_contents,cell_typ):
                     if parse_cols is None or should_parse[j]:
                         row.append(_parse_cell(value,typ))
                 data.append(row)
-    
+
+            if sheet.nrows == 0:
+                return DataFrame()
+
             if header is not None:
```

Comment:
what about ``sheet.nrows==1`` (and its the header)? do we construct an empty frame with the correct columns?